### ISSUE TITLE
Accessibility: pins need alt-text

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-hotgraphic#59]
When running JAWS: If I tab to a hot pin there is no indication I am on a pin or that I should open it. 

Each pin should have its own alt text to instruct users.
